src/components/Modals/SpendLLDModal.js

```diff
@@ -0,0 +1,152 @@
+import ModalRoot from './ModalRoot';
+import { TextInput } from '../InputComponents';
+import Button from '../Button/Button';
+import InputSearch from '../InputComponents/InputSearchAddressName';
+
+// STYLES
+import styles from './styles.module.scss';
+import { parseDollars, isValidSubstrateAddress } from '../../utils/walletHelpers';
+import { congressActions } from '../../redux/actions';
+import { congressSelectors } from '../../redux/selectors';
+import Validator from '../../utils/validator';
+import useCongressExecutionBlock from '../../hooks/useCongressExecutionBlock';
+
+function SpendLLDModal({ closeModal }) {
+  const dispatch = useDispatch();
+  const balances = useSelector(congressSelectors.balances);
+  const balance = new BN(balances?.liquidAmount?.amount ?? 0);
+
+  const {
+    handleSubmit,
+    formState: { errors },
+    register,
+    setValue,
+    watch,
+  } = useForm({
+    mode: 'all',
+    defaultValues: {
+      votingDays: '7',
+    },
+  });
+  const votingDays = watch('votingDays');
+  const executionBlock = useCongressExecutionBlock(votingDays);
+
+  const transfer = (values) => {
+    dispatch(
+      congressActions.congressSendLld.call({
+        transferToAddress: values.recipient,
+        transferAmount: parseDollars(values.amount),
+        remarkInfo: values.description || 'Congress spend LLD',
+        executionBlock,
+      }),
+    );
+    closeModal();
+  };
+
+  return (
+    <form
+      className={styles.getCitizenshipModal}
+      onSubmit={handleSubmit(transfer)}
+    >
+      <div className={styles.h3}>Spend LLD</div>
+      <div className={styles.description}>
+        You are going to create spend token proposal
+      </div>
+
+      <div className={styles.title}>Spend to address</div>
+      <InputSearch
+        errorTitle="Recipient"
+        register={register}
+        name="recipient"
+        placeholder="Spend to address"
+        isRequired
+        setValue={setValue}
+        validate={(v) => {
+          if (!isValidSubstrateAddress(v)) return 'Invalid Address';
+          return true;
+        }}
+      />
+      {errors?.recipient?.message && (
+        <div className={styles.error}>{errors.recipient.message}</div>
+      )}
+
+      <div className={styles.title}>Amount LLD</div>
+      <TextInput
+        errorTitle="Amount LLD"
+        register={register}
+        validate={(textValue) => Validator.validateDollarsValue(balance, textValue)}
+        name="amount"
+        placeholder="Amount LLD"
+        required
+      />
+      {errors?.amount?.message && (
+        <div className={styles.error}>{errors.amount.message}</div>
+      )}
+
+      <div className={styles.title}>Spend description</div>
+      <TextInput
+        register={register}
+        minLength={{ value: 3, message: 'Min description length is 3 chars' }}
+        maxLength={{ value: 256, message: 'Min description length is 256 chars' }}
+        name="description"
+        placeholder="Spend description"
```

Comment:
why are spendLLMModal and spendAssetModal different modals? 

I understand for spendPolitipooledLLMModal, but thats different